web_external/js/views/body/SubmissionView.js

```diff
@@ -56,7 +95,8 @@ covalic.views.SubmissionView = covalic.View.extend({
                             this.phase.getAccessLevel() < girder.AccessType.WRITE),
             JobStatus: girder.jobs_JobStatus,
             job: this.job,
-            created: girder.formatDate(this.submission.get('created'), girder.DATE_SECOND)
+            created: girder.formatDate(this.submission.get('created'), girder.DATE_SECOND),
+            download: this.phase.getAccessLevel() > girder.AccessType.READ
```

Comment:
Since everything in javascript is a floating point number, maybe we should be doing this comparisons with an epsilon 😰 

j/k